### PR TITLE
Added a gradient to the gauge.

### PR DESCRIPTION
**Pull request by abecam**
Also reduced the minimum size (for my needs).
The minified CSS is not updated, but I updated the example html as it needs 2 more divs.
It could be a different variant, so feel free to do whatever you like with it ;)

### CODE REVIEW

File: gauge.css [hunk 1]
```diff
@@ -78,6 +92,36 @@
...
+    top: 10%;
+    left: 10%;
+    width: calc(80%);
+    height: calc(80%);
+    border-left: 0em solid;
+    border-top: 0em solid;
+    border-right: 0em solid;
+    border-bottom: 0em solid transparent;
+    border-radius: 50%;
+    //background-image: linear-gradient(to right, #c0392b 0%, #f1c40f 50%, #1abc9c 100%);
```
Comment (rotvalli) [inline gauge.css]: Dead code

File: gauge.css [hunk 2]
```diff
@@ -78,6 +92,36 @@
...
+    top: 15%;
+    left: 15%;
+    width: calc(70%);
+    height: calc(70%);
+    border-left: 0em solid;
+    border-top: 0em solid;
+    border-right: 0em solid;
+    border-bottom: 0em solid transparent;
+    border-radius: 50%;
+    //background-image: linear-gradient(to right, #c0392b 0%, #f1c40f 50%, #1abc9c 100%);
```
Comment (rotvalli) [inline gauge.css]: Dead code

File: example.html [hunk 3]
```diff
@@ -6,7 +6,7 @@
     <title>Pure CSS gauge v0.2</title>
     <meta name="description" content="Pure CSS gauge">
     <meta name="author" content="Otto Salminen">
-    <link rel="stylesheet" href="gauge.min.css?v=0.2">
```
Comment (rotvalli) [inline example.html]: Would use the minified version on the default example

Comment (abecam) [reply example.html]: Minified version updated and reverted to it here :)

File: gauge.css (hunk 2 above)
Comment (abecam) [reply gauge.css]: Removed

File: gauge.css (hunk 1 above)
Comment (abecam) [reply gauge.css]: Removed, sorry about that.

**Review by rotvalli — APPROVED** [review]

**Comment by rotvalli** [comment]
Thanks for your variant. I added it to the examples  https://cssgauge.peukalo.net/

**Pull request closed without merging**